apps/main-cf.yml.j2

```
@@ -154,6 +154,10 @@ Resources:
     Properties:
       Parameters:
         ComputeEnvironmentArn: !GetAtt Cluster.Outputs.ComputeEnvironmentArn
+        {% for job_type, job_spec in job_types.items() if 'Default' != job_spec['compute_environment']['name'] %}
```

I think I'd actually prefer to have the default be to leave `compute_environment` out of the job spec entirely, and then these loops all look like:
```suggestion
        {% for job_type, job_spec in job_types.items() if 'compute_environment' in job_spec %}
```
which just keeps the job spec slightly less cluttered. But I could probably be convinced otherwise if there's a compelling reason to always include it in the job spec.

---

I really prefer imperative over implied. We also specify a default cost of 1.0 in all the files, which probably isn't necessary, but I find it helpful. At the very least, it makes it impossible to miss that its an option when using an existing job_spec as an example (since this isn't documented).

---

Yep, that's fair. If it were documented separately, I'd argue for leaving it out, but keeping it as a visible option in the absence of documentation is a good reason for leaving it in.